lib/cliCommon.js

```diff
@@ -24,7 +24,19 @@ function printCliResultError (error) {
     console.log(messages.submitGithubIssue);
 }
 
+/**
+ * @param {Error} error
+ * @param {Array<{message: string}>} [error.messages]
```

Comment:
there is a single input param. Why are we listing `error.message` seperately here ?

Reply:
Lazy copy paste :P